bundle-preview/bundle-ai-chat.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div id="ai-chat-container"></div>
+    <script type="module">
+      import * as aiChat from "./static/bundles/aiChat.es.js"
```

Comment:
@asadali145 btw: With the module script tag, this can also be imported from a CDN like
```js
import * as aiChat from "https://unpkg.com/@mitodl/smoot-design@0.0.0-045ab6a/dist/bundles/aiChat.es.js"
```
_That's probably not a CDN we should use in production, but good for testing._

Maybe would be good way for xblock to consume?

Reply:
Mentioning:
1. The `type="module"` version worked fine on Live, but not in CMS (Studio) or Preview LMS, at least on residential mitx. I'm not sure why.
2. The UMD version (global variable) also did not work in Studio or Preview. This seems ot be because Studio and Preview are adding the script tags asynchronously after some API requests, whereas the script tags are included in the initial document HTML on live site. When the script tags are added async, their execution order is not  guaranteed, so the initialization run prior to the bundle download finishing.

For now we solved via doing `import("...aiChat.es.js").then(module => { /* init code */ }), which works in all three environments.